protocol/agents.json

```diff
@@ -82,19 +82,29 @@
             ],
             "type": "os"
         },
+        {
+            "identifier": "darwin",
```

Comment:
How does Darwin differ from macOS? Surely they're one and the same< which is to say I don't think we need to add Darwin.

Reply:
Updated to `macOS-iOS`

Reply:
Let's stick with `darwin`. I think it may still be of use as from what I can tell we can probably infer macOS version from Darwin version when we look at logs later on (see [here](https://en.wikipedia.org/wiki/Darwin_(operating_system)#Release_history)).

Reply:
Done !